metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -429,9 +429,16 @@ class Compilers(
     }
 
     created.getOrElse {
-      val sourcesWithJdk = JdkSources(userConfig().javaHome) match {
-        case Some(absPath) => absPath.toNIO :: sources
-        case None => sources
+      val sourcesWithScala =
+        if (sources.filter(_.toString.contains("scala-library")).isEmpty)
```

Comment:
Could we move the logic to `StandaloneSymbolSearch`? We already do download scala library there and it would be good to make sure that both `apply` work correctly.

Reply:
Sure, I can move it. Do you think I should do the same for the JDK sources as well then?

Reply:
Makes sense! Just in case someone creates the StandaloneSymbolSearch in a different place we will not need to fix it all again.

Reply:
Sounds good! Moved both.